### PR TITLE
fix: prevent manual addition of duplicate member IDs in on-chain groups

### LINKED ISSUE #302: Dashboard: Add a member manually in on-chain groups only if it was not added before

### PR DESCRIPTION
**Pull request by tanyas-codes**
Previously, on-chain groups did not properly filter duplicate member IDs, allowing the same ID to be added multiple times without restriction. Now, when a user attempts to add an existing member ID to an on-chain group, an alert message  with the exact id/id's is displayed to notify the user and prevent the duplicate addition.

This resolves #302 

## Description

Alerting the user when a member id that is already present is added manually .
-Alert message with the exact duplicate id/id's is shown 

Additionally there was another bug that i came across while solving,the user can enter hexa values 
-it is saved in decimal format on the chain
-now another same hexa code can be added ,leads to duplicacy 

-Thus hexa checking is additionally added,it filters and converts to decimal and compares with the stored member id's

## Does this introduce a breaking change?

-   [ ] No

### CODE REVIEW

**Comment by tanyas-codes** [comment]
@vplasencia Hey i've made the pr ! check it out !

**Comment by vplasencia** [comment]
Thank you very much @tanyas-codes 

I will check them out today.

**Review by vplasencia — COMMENTED** [review]
Hey @tanyas-codes 

What about using BigInt to check? It should be simpler because you won't need to check if it's hexadecimal or not since`BigInt("1") === BigInt("0x1")`

Let me know what you think.

**Comment by tanyas-codes** [comment]
I was concerned with its compatibility with older browsers ! although,

Yes! That will make the code simpler and concise ! ->I will make another commit with the changes required.

**Review by vplasencia — COMMENTED** [review]
It looks great ✨

Just added some suggestions.

File: apps/dashboard/src/components/add-member-modal.tsx [hunk 1]
```diff
@@ -71,6 +71,33 @@ export default function AddMemberModal({
             alert("Please ensure there are no repeated member IDs!")
             return
         }
+        if (group.type === "on-chain" && group.members) {
+            const existingMembers = new Set(
+                group.members.map((memberId) =>
+                    typeof memberId === "string" ? BigInt(memberId) : memberId
```
[the remark this answers is not in the document]
Comment (tanyas-codes) [reply apps/dashboard/src/components/add-member-modal.tsx:77]: yes ,memberId as per the parseMemberIds.ts file  has defined the input as a string in the utils folder ,i'll make this change !

**Review by vplasencia — APPROVED** [review]
Great work ✨

Thank you so much @tanyas-codes

**Review by 0xjei — COMMENTED** [review]
LGTM!

**Review by 0xjei — APPROVED** [review]

**Pull request merged**